.pylintrc

```diff
@@ -68,14 +68,14 @@ disable=missing-docstring,
       duplicate-code,
       ungrouped-imports, # Leave this up to isort
       wrong-import-order, # Leave this up to isort
-      bad-continuation, # Leave this up to black
       line-too-long, # Leave this up to black
       exec-used,
       super-with-arguments, # temp-pylint-upgrade
       isinstance-second-argument-not-valid-type, # temp-pylint-upgrade
       raise-missing-from, # temp-pylint-upgrade
       unused-argument, # temp-pylint-upgrade
       redefined-builtin,
+      cyclic-import,
```

Comment:
Imports like this should be fixed https://github.com/open-telemetry/opentelemetry-python/blob/36aaa84d17284de6c873eadc0b955d97fe06b07b/opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py#L28-L29